support-workers/src/main/scala/com/gu/support/workers/lambdas/SendThankYouEmail.scala

```diff
@@ -53,8 +51,8 @@ class SendThankYouEmail(
 
     for {
       emailFields <- emailBuilder.buildEmail(state)
-      emailResult <- Future.sequence(emailFields.map(emailService.send))
-    } yield HandlerResult(emailResult, requestInfo)
+      _ = emailFields.map(emailService.send)
+    } yield HandlerResult((), requestInfo)
```

Comment:
nothing was using the actual SendMessageResult, so I made it return Unit instead